src/llmcompressor/modifiers/awq/base.py

```diff
@@ -401,11 +403,9 @@ def cache_smooth_activations_hook(
                 args: Tuple[torch.Tensor, ...],
                 _output: torch.Tensor,
             ):
-                # Assume that first argument is the input
-                inp = args[0].cpu().detach().squeeze()
-
                 self._smooth_activation_means[smooth_name] = _accumulate_mean(
-                    inp,
+                    # Assume that first argument is the input
+                    args[0].cpu().detach().squeeze(),
```

Comment:
Do we actually need to move to cpu device? Isn't it better to keep on the module device?

Reply:
Do we actually need to squeeze? I think we should make the `_accumulate_mean` function handle shape logic, if necessary